Features/RoleUI/RoleUI.feature

```diff
@@ -0,0 +1,166 @@
+Feature: Create, delete, update and View the Role UI
+  In order to create, delete, update and view the ROLE UI
+  As an Administrator user
+  I need to be able to access and navigate through the Role UI interface
+
+  Background: 
+    Given I am logged in as an Administrator in PlatformUI
+
+  ##
+  #
+  # Validation Roles fields
+  #
+  ##
+  @javascript @common
+  Scenario: Verify the existence of Roles page and its content
+    Given I am on the Home page
+    And I have an "Organizers" role
+    When I click on the Administration Dashboard Navigation Item
+    And I click on the "Roles" link
+    Then I see the Roles page
+    And I see the following roles:
+      | Name          | ID |
+      | Anonymous     | 1  |
+      | Administrator | 2  |
+      | Editor        | 3  |
+      | Partner       | 4  |
+      | Member        | 5  |
+      | Organizers    | 6  |
+    And I see an "Edit" button for each one of the roles
+    And I see a "Create" button
+
+  @javascript @common
+  Scenario: Verify the information displayed by one of the roles
+    Given I am on the Roles page
+    When I click on the "Anonymous" link
+    Then I see the following limitations fields:
+      | Module | Function | Limitation |
+    And I should see an "Edit" button
+    And I should see a "Delete" button
+    And I should see a label for the Role Assigments
+    And I should see a group with the Role Assigments
+      | User/group | Limitation |
+
+  @javascript @edge
+  Scenario: Display the information of a role with empty policies
+    Given I have an "Organizers" role
```

Comment:
This sentence is not clear, when I first read it I assumed it meant Administrator had this role (assigned). Because `I am an Administrator`, so `I have <role>` basically means it is assigned to me :)

But what is being implied here from the use is that it exist, priorly created, so maybe rephrase this sentence to more clearly relfect that.